.gitignore

```diff
@@ -10,4 +10,7 @@ node_modules
 dist
 *.zip
 
-.stellar
\ No newline at end of file
+.vscode/*
```

Comment:
This directory contains the VS Code workspace settings, and is intended to be in the repository.

Reply:
The problem with committing the `.vscode` folder is it means none of those settings can be overwritten locally since they are "stronger" than user level settings. 
This can cause issues when settings differ from user to user. (example: Your vscode eslint seems to function different than mine locally, and combined with "formatOnSave" causing my linter to unindent the .json and html files every time I touch them) 
<img width="433" alt="Screen Shot 2021-01-19 at 5 56 40 AM" src="https://user-images.githubusercontent.com/7035467/105032325-3fbf0700-5a1c-11eb-806f-fdccc243d83d.png">

I've put back in `.vscode/settings.json.default` So the settings can be inherited if needed. The eslint stuff really doesn't need to be forced on each user on save, since the pipeline should be configured to catch lint errors, and the actual formatting can be handled via the command line :fix actions.

Reply:
I'll have to put some thought into this. I do realize that I did not include the "recommended extensions," which allow these settings to work as intended. I think that I'd prefer, if you are using VS Code already, that files be saved the exact same way without a manual step that could be forgotten (besides the initial extension setup).

Reply:
I like the workaround